### ISSUE TITLE
Pass Command-Line Flags Failing

### ISSUE THREAD

**Issue by mastermindg** [bitnami/bitnami-docker-mariadb#122]
-------------------------------
    BUG REPORT INFORMATION
-------------------------------

**Description**

I'm trying to pass command-line arguments to MySqlD on startup. This was brought up on https://github.com/bitnami/bitnami-docker-mariadb/issues/14 and patched in 2015 but is no longer working for me with the latest image.

**Steps to reproduce the issue:**

```
docker run --name mariadb -e ALLOW_EMPTY_PASSWORD=yes bitnami/mariadb:latest mysqld --max-connect-errors=1000
```

**Describe the results you received:**

```
2017-05-12 13:08:23 140297585899392 [Note] mysqld (mysqld 10.1.23-MariaDB) starting as process 13 ...
mysqld: Please consult the Knowledge Base to find out how to run mysqld as root!
2017-05-12 13:08:23 140297585899392 [ERROR] Aborting
**Describe the results you expected:**
```

MySQLD should start in the container with the argument that was passed to it.

- Output of `docker version`:

```
Docker version 1.13.1, build 092cba3
```

- Output of `docker info`:

```
Containers: 7
 Running: 3
 Paused: 0
 Stopped: 4
Images: 66
Server Version: 1.13.1
Storage Driver: overlay
 Backing Filesystem: extfs
 Supports d_type: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: aa8187dbd3b7ad67d8e5e3a15115d3eef43a7ed1
runc version: 9df8b306d01f59d3a8029be411de015b7304dd8f
init version: 949e6fa
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.8.0-51-generic
Operating System: Ubuntu 16.10
OSType: linux
Architecture: x86_64
CPUs: 8
Total Memory: 15.53 GiB
Name: myserver
ID: UZBV:QJN5:XJKT:5PY6:C63W:PV4Z:K7OK:5KXZ:IH36:5KYK:36WN:Q4QA
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Username: mastermindg
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Comment by prydonius** [comment]
Hey @mastermindg,

Looks like we dropped support for this somewhere along the way, however it shouldn't be too much work to add this back in. The [startup script](https://github.com/bitnami/bitnami-docker-mariadb/blob/master/10/rootfs/run.sh) would need to be run directly in the [entrypoint](https://github.com/bitnami/bitnami-docker-mariadb/blob/master/10/rootfs/app-entrypoint.sh#L15) and extra arguments could be passed to the run.sh script.

**Comment by mastermindg** [comment]
The entrypoint would have to be changed so that run.sh could be run directly from app-entrypoint.sh right?

**Comment by prydonius** [comment]
@mastermindg yep app-entrypoint.sh should run run.sh and pass it the args (CMD) provided to app-entrypoint.sh

**Comment by pdhimal1** [comment]
@mastermindg @prydonius has this been resolved?

**Comment by arapulido** [comment]
@tompizmor Tomás, while you work on the changes for the mariadb container, maybe you can tackle this one as well, please?

**Comment by tompizmor** [comment]
yep, I can work on this!

**Comment by tompizmor** [comment]
Hi @pdhimal1,

We have just released MariaDB version 10.1.28-r1 which allows specifying command line-flags via the `MARIADB_EXTRA_ARGS` env var. Please take a look at this section in the README:

https://github.com/bitnami/bitnami-docker-mariadb#passing-extra-command-line-flags-to-mysqld-startup

**Comment by pdhimal1** [comment]
@tompizmor Thank you!
I can confirm that this works in 10.1.28-r1
